package/network/utils/wireguard-tools/files/wireguard.sh

```diff
@@ -42,6 +44,11 @@ proto_wireguard_setup_peer() {
 	config_get endpoint_port "${peer_config}" "endpoint_port"
 	config_get persistent_keepalive "${peer_config}" "persistent_keepalive"
 
+	if [ "${disabled}" -eq 1 ]; then
+		# skip disabled peers
+		return
```

Comment:
Since the if directly below returns an integer status, this should probably return one, too.

Reply:
I added `0` to both `return`s.